packages/server/src/asyncQuestion/asyncQuestion.controller.ts

```diff
@@ -181,14 +180,14 @@ export class asyncQuestionController {
       });
 
       if (usersInCourse?.length) {
-        await getRepository(UserCourseAsyncQuestionModel)
+        await getRepository(UnreadAsyncQuestionModel)
           .createQueryBuilder()
```

Comment:
```suggestion
      await UnreadAsyncQuestionModel.createQueryBuilder()
```
deprecated syntax, getRepository(x) is deprecated in later versions of typeorm

Reply:
That's good to know.